frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -118,10 +109,39 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
     const usableConfigs = globalConfigs
       .filter((item) => this.checkFlags(item))
       .map((item) => item.properties);
-    const allItems = usableConfigs.length > 0 && items.concat(usableConfigs);
-    const sortedItems =
-      usableConfigs.length > 0 ? _.sortBy(_.flatten(allItems), 'kind', 'asc') : items;
-
+    let allItems = items.concat(usableConfigs);
+    allItems = allItems
+      .map((item) => {
+        const apiExplorerLink = `/api-resource/cluster/${referenceForModel(item.model)}`;
+        const resourceLink = resourcePathFromModel(item.model, item.name, item.namespace);
+        return {
+          ...item,
+          resourceLink,
+          menuItems: [
+            editYAMLMenuItem(item.kind, resourceLink),
+            viewAPIExplorerMenuItem(item.kind, apiExplorerLink),
+            ...(item.kind === 'OAuth' ? oauthMenuItems : []),
+          ],
+          description: getResourceDescription(item.model),
+        };
+      })
+      .concat({
+        kind: 'Alertmanager',
+        uid: 'alertmanager',
```

Comment:
This isn't a UID. Let's just call it `id`

```suggestion
        id: 'alertmanager',
```